design-tokens/tokens/layer1/base.json

```diff
@@ -428,10 +428,10 @@
           }
         },
         "large": {
-          "regular": {
+          "medium": {
             "value": {
               "fontFamily": "{echoes.fontFamily.sans}",
-              "fontWeight": "{echoes.fontWeight.regular}",
+              "fontWeight": "{echoes.fontWeight.medium}",
               "lineHeight": "{echoes.lineHeight.30}",
```

Comment:
Why is `regular` disappearing from `large` ?

Reply:
🤦‍♂️ 

It should be fixed now